tests/e2e/airgap_test.go

```diff
@@ -0,0 +1,229 @@
+			Expect(err).To(Not(HaveOccurred()))
+		})
+
+		By("Installing CertManager", func() {
+			// Set flags for cert-manager installation
+			flags := []string{
+				"upgrade", "--install", "cert-manager", "/opt/rancher/helm/cert-manager-" + certManagerVersion + ".tgz",
+				"--namespace", "cert-manager",
+				"--create-namespace",
+				"--set", "image.repository=rancher-manager.test:5000/cert/cert-manager-controller",
+				"--set", "webhook.image.repository=rancher-manager.test:5000/cert/cert-manager-webhook",
+				"--set", "cainjector.image.repository=rancher-manager.test:5000/cert/cert-manager-cainjector",
+				"--set", "startupapicheck.image.repository=rancher-manager.test:5000/cert/cert-manager-ctl",
+				"--set", "installCRDs=true",
+				"--wait", "--wait-for-jobs",
+			}
+
+			RunHelmCmdWithRetry(flags...)
+
+			checkList := [][]string{
+				{"cert-manager", "app.kubernetes.io/component=controller"},
+				{"cert-manager", "app.kubernetes.io/component=webhook"},
+				{"cert-manager", "app.kubernetes.io/component=cainjector"},
+			}
+			err := rancher.CheckPod(k, checkList)
+			Expect(err).To(Not(HaveOccurred()))
+		})
+
+		By("Installing Rancher", func() {
+			// TODO: Use the DeployRancherManager function from install.go
+			rancherAirgapVersion, err := exec.Command("bash", "-c", "ls /opt/rancher/helm/rancher-*.tgz").CombinedOutput()
+			// Set flags for Rancher Manager installation
+			flags := []string{
+				"upgrade", "--install", "rancher", string(rancherAirgapVersion),
+				//"upgrade", "--install", "rancher", "/opt/rancher/helm/rancher-" + rancherHeadVersion + ".tgz",
+				"--namespace", "cattle-system",
+				"--create-namespace",
+				"--set", "hostname=rancher-manager.test",
+				"--set", "extraEnv[0].name=CATTLE_SERVER_URL",
+				"--set", "extraEnv[0].value=https://rancher-manager.test",
+				"--set", "extraEnv[1].name=CATTLE_BOOTSTRAP_PASSWORD",
+				"--set", "extraEnv[1].value=rancherpassword",
+				"--set", "replicas=1",
+				"--set", "useBundledSystemChart=true",
+				"--set", "rancherImage=rancher-manager.test:5000/rancher/rancher",
+				"--set", "systemDefaultRegistry=rancher-manager.test:5000",
+			}
+
+			RunHelmCmdWithRetry(flags...)
+
+			// Wait for all pods to be started
+			checkList := [][]string{
+				{"cattle-system", "app=rancher"},
+				{"cattle-fleet-local-system", "app=fleet-agent"},
+				{"cattle-system", "app=rancher-webhook"},
+			}
+			err = rancher.CheckPod(k, checkList)
+			Expect(err).To(Not(HaveOccurred()))
+		})
+
+		By("Installing Elemental Operator", func() {
+			// Install Elemental Operator CRDs first
+			// Set flags for Elemental Operator CRDs installation
+			flags := []string{
+				"upgrade", "--install", "elemental-crds", "/opt/rancher/helm/elemental-operator-crds-chart-" + elementalVersion,
+				"--namespace", "cattle-elemental-system",
+				"--create-namespace",
+			}
+
+			RunHelmCmdWithRetry(flags...)
+			time.Sleep(20 * time.Second)
```

Comment:
Maybe better to check the status of the deployment/pod, for a next change, PR is big enough ;-)